### PR TITLE
Added lab3 implementation

### PR DESCRIPTION
**Pull request by maksimkha**
u can also check using this link https://agrexa-60b32.web.app/index.html

### CODE REVIEW

File: Lab3/public/account.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+          </div>
+      </div>
+    </header>
+    <main>
+      <div class="pagebar">
+        <h1>Account
+      </div>
+      <form class="account-form">
+        <div class="line-container">
+            <label>Name:</label>
```
Comment (gmltA) [inline Lab3/public/account.html:34]: Label for what?

File: Lab3/public/item.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      <div class="topbar">
+        <div>
+          <label class="switch">
+            <input type="checkbox" id="togBtn">
+            <div class="slider round"></div>
+          </label>
+        </div><!-- 
+        --><div>
+            <form>
+              <button class="topbar__site-name" formaction="index.html">Agrexa</button>
```
Comment (gmltA) [inline Lab3/public/item.html:18]: That's not how you should make a link.

File: Lab3/public/styles.css [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+    width: 80vw;
+    background-color:#674d3c;
+    margin: auto;
+    margin-top: 1vh;
+    padding: 1vh 1vh;
+    border-radius: 25px;
+}
+.topbar > div{
+    display: inline-block;
+    zoom: 1;
```
Comment (gmltA) [inline Lab3/public/styles.css:14]: ?

File: Lab3/public/styles.css [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+    border-radius: 50%;
+}
+
+input:checked + .slider {
+    background-color: #e0876a;
+}
+
+input:checked + .slider:before {
+    -webkit-transform: translateX(26px);
+    -ms-transform: translateX(26px);
```
Comment (gmltA) [inline Lab3/public/styles.css:75]: ?

**Review by gmltA — CHANGES_REQUESTED** [review]

File: Lab3/public/item.html (hunk 2 above)
Comment (maksimkha) [reply Lab3/public/item.html]: yeap, changed 'form' to 'a'

File: Lab3/public/styles.css (hunk 3 above)
Comment (maksimkha) [reply Lab3/public/styles.css]: unnecessary zoom is removed

File: Lab3/public/styles.css (hunk 4 above)
Comment (maksimkha) [reply Lab3/public/styles.css]: fixed

File: Lab3/public/account.html (hunk 1 above)
Comment (maksimkha) [reply Lab3/public/account.html:32]: can't get this error. i have labels for labeling my input fields. am i doing smthing wrong?

Comment (gmltA) [reply Lab3/public/account.html:32]: Your labels are currently labeling nothing because you have to link between them and corresponding inputs.

**Review by gmltA — CHANGES_REQUESTED** [review]
And now you also have to tell me where this `zoom: 1` came from.

**Comment by maksimkha** [comment]
got it, so i changed my acc page. and those 'zoom: 1' were like a bunch of brainless ideas of trying to adapt my views to both screen sizes, forgot to remove those

**Review by gmltA — APPROVED** [review]
That's a <kbd>6</kbd>.

**Pull request merged**